homeassistant/components/airthings_ble/sensor.py

```diff
@@ -107,9 +115,66 @@
 }
 
 
+@callback
+def async_migrate(hass: HomeAssistant, address: str, sensor_name: str):
+    """Migrate entities to new unique ids (with BLE Address)."""
+
+    ent_reg = entity_async_get(hass)
+
+    device_registry = device_async_get(hass)
+    entity_registry = entity_async_get(hass)
+
+    device = device_registry.async_get_device(identifiers={(DOMAIN, address)})
+
+    if not device:
+        return
```

Comment:
We can lookup the device by connection and verify its the right config entry instead of adding the identifiers

Reply:
Was this what you asked for..? Or did I miss something? I changed from `identifiers` to `connections`.

Reply:
All good now 👍